.github/workflows/ci.yml

```diff
@@ -1368,7 +1368,7 @@ ${{ hashFiles('.pre-commit-config.yaml') }}"
         with:
           python-version: ${{ env.PYTHON_MAJOR_MINOR_VERSION }}
       - name: "Free space"
-        run: ./scripts/ci/tools/free_space.sh
+        run: ./scripts/ci/tools/free-space.py
```

Comment:
`free-space` is an invalid module name, which could make development difficult in the future if we wanted to reuse the code snippet.

Reply:
`freespace` or `free_space` could be good names for a module? 
I found it this: "Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability."

Reply:
I am calling it: freespace.py